### PR TITLE
Polish translation update part1

### PR DESCRIPTION
**Pull request by zutmkr**
First part of translation. Lets review it together and see if we can make it better. 

Calling @qndel for help.
Please add others reviewers if you know they can contribute.

### CODE REVIEW

File: Translations/pl.po [hunk 1]
```diff
@@ -391,48 +366,44 @@ msgstr "Anuluj"
...
 
 #: Source/DiabloUI/selconn.cpp:68
 msgid "Change Gateway"
-msgstr ""
+msgstr "Zmień Bramkę"
 
 #: Source/DiabloUI/selconn.cpp:101
 msgid "All computers must be connected to a TCP-compatible network."
-msgstr ""
+msgstr "Wszystkie komputery muszą być połączone siecią kompatybilną z TCP."
```
Comment (qndel) [inline Translations/pl.po:398]:
```suggestion
msgstr "Wszystkie komputery muszą być połączone z siecią kompatybilną z TCP."
```

File: Translations/pl.po [hunk 2]
```diff
@@ -747,68 +720,72 @@ msgstr "Nie"
...
 
 #: Source/DiabloUI/support_lines.cpp:19
 msgid "  DevilutionX is not supported or maintained by Blizzard Entertainment,"
-msgstr ""
+msgstr "DevilutionX nie jest wspierane ani przez Blizzard Entertainment"
 
 #: Source/DiabloUI/support_lines.cpp:20
-msgid "  nor GOG.com. Neither Blizzard Entertainment nor GOG.com has tested"
-msgstr ""
+msgid "  nor GOG.com. Neither Blizzard Entertainment or GOG.com has not tested"
```
Comment (qndel) [inline Translations/pl.po:762]: to cofnij

File: Translations/pl.po [hunk 3]
```diff
@@ -1295,52 +1265,52 @@ msgstr "Zapisywanie..."
...
 #: Source/error.cpp:71
 msgid "Those who defend seldom attack"
-msgstr "Atak nie zawsze jest najlepszą obroną."
+msgstr "Atak nie zawsze jest najlepszą obroną"
 
 #. TRANSLATORS: Shrine Text. Keep atmospheric. :)
 #: Source/error.cpp:72
 msgid "The sword of justice is swift and sharp"
-msgstr "Miecz sprawiedliwości ma jedno ostrze."
+msgstr "Miecz sprawiedliwości jest szybki i ostry!"
```
Comment (qndel) [inline Translations/pl.po:1283]:
```suggestion
msgstr "Miecz sprawiedliwości jest szybki i ostry"
```

File: Translations/pl.po [hunk 4]
```diff
@@ -1350,42 +1320,42 @@ msgstr "Tajemna moc przynosi zniszczenie."
...
 #: Source/error.cpp:84
 msgid "Wherever you go, there you are"
-msgstr "Pójdź w nieznane."
+msgstr "Gdziekolwiek pójdziesz, tam jesteś"
 
 #. TRANSLATORS: Shrine Text. Keep atmospheric. :)
 #: Source/error.cpp:85
 msgid "Energy comes at the cost of wisdom"
-msgstr "Energia zyskana kosztem mądrości."
+msgstr "Energia jest kosztem mądrości"
```
Comment (qndel) [inline Translations/pl.po:1348]:
```suggestion
msgstr "Energia kosztem mądrości"
```

File: Translations/pl.po [hunk 5]
```diff
@@ -1350,42 +1320,42 @@ msgstr "Tajemna moc przynosi zniszczenie."
...
 #: Source/error.cpp:86
 msgid "Riches abound when least expected"
-msgstr "Bogactwo przybywa nieoczekiwanie."
+msgstr "Bogactwo przybywa nieoczekiwanie"
 
 #. TRANSLATORS: Shrine Text. Keep atmospheric. :)
 #: Source/error.cpp:87
 msgid "Where avarice fails, patience gains reward"
-msgstr "Chciwość zawodzi, cierpliwość popłaca."
+msgstr "Chciwość zawodzi, cierpliwość popłaca"
```
Comment (qndel) [inline Translations/pl.po:1358]:
```suggestion
msgstr "Gdzie chciwość zawodzi, tam cierpliwość popłaca"
```

File: Translations/pl.po [hunk 6]
```diff
@@ -1793,18 +1765,19 @@ msgstr "ESC - wyjście, Strzałki - przewijanie"
 
 #: Source/init.cpp:214
 msgid "Some Hellfire MPQs are missing"
-msgstr ""
+msgstr "Brakuje reszty plików MPQ do Hellfire"
```
Comment (qndel) [inline Translations/pl.po:1768]:
```suggestion
msgstr "Brakuje części plików MPQ do Hellfire"
```

File: Translations/pl.po [hunk 7]
```diff
@@ -1793,18 +1765,19 @@ msgstr "ESC - wyjście, Strzałki - przewijanie"
...
 "Please copy all the hf*.mpq files."
 msgstr ""
+"Nie znaleziono wszystkich plików MPQ do Hellfire.\n"
+"Skopiuj wszystkie pliki hf*.mpq do folderu."
 
 #: Source/init.cpp:224
-#, fuzzy
 msgid "Unable to create main window"
-msgstr "Nie udało się stworzyć postaci."
+msgstr "Nie udało się utworzyć nowego okna"
```
Comment (qndel) [inline Translations/pl.po:1780]:
```suggestion
msgstr "Nie udało się utworzyć głównego okna"
```

File: Translations/pl.po [hunk 8]
```diff
@@ -3213,7 +3186,7 @@ msgid "The Defender"
 msgstr "Obrońca"
 
 #: Source/itemdat.cpp:418
-msgid "Gryphon's Claw"
+msgid "Gryphons Claw"
```
Comment (qndel) [inline Translations/pl.po:3189]: cofnij zmianę

File: Translations/pl.po [hunk 9]
```diff
@@ -3377,7 +3350,7 @@ msgid "Rod of Onan"
 msgstr "Drąg Onana"
 
 #: Source/itemdat.cpp:459
-msgid "Helm of Spirits"
+msgid "Helm of Sprits"
```
Comment (qndel) [inline Translations/pl.po:3353]: też cofnij

**Review by qndel — CHANGES_REQUESTED** [review]

File: Translations/pl.po (hunk 2 above)
Comment (zutmkr) [inline Translations/pl.po:762]:
```suggestion
msgid "  nor GOG.com. Neither Blizzard Entertainment nor GOG.com has tested"
```

File: Translations/pl.po (hunk 9 above)
Comment (zutmkr) [inline Translations/pl.po:3353]:
```suggestion
msgid "Helm of Spirits"
```

File: Translations/pl.po (hunk 8 above)
Comment (zutmkr) [inline Translations/pl.po:3189]:
```suggestion
msgid "Gryphon's Claw"
```

**Comment by AJenbo** [comment]
:+1:

**Comment by AJenbo** [comment]
@zutmkr if you have time to update the translation now would be a good time to make it for the next release :)

There are about 250 lines that needs an update.

**Pull request merged**